crates/core/tedge_api/src/device_profile.rs

```diff
@@ -39,20 +39,28 @@ impl CommandPayload for DeviceProfileCmdPayload {
 #[derive(Debug, Deserialize, Serialize, Eq, PartialEq, Clone)]
 #[serde(rename_all = "camelCase")]
 pub struct DeviceProfileOperation {
-    operation: OperationType,
-    skip: bool,
     #[serde(flatten)]
-    payload: OperationPayload,
+    pub operation: Operation,
+    pub skip: bool,
```

Comment:
I just noticed that `@skip` is a mandatory field now. It should have been optional. If that's a regression, it must be fixed. But, if the previous versions also had the same limitation, it can be addressed separately by creating a dedicated ticket.

Reply:
yes, that is regression and it was like that since the beginning. I would address that in separate PR.